config/php-cs-fixer.php

```diff
@@ -41,6 +41,10 @@
 
             // function notation
             'native_function_invocation' => ['include' => ['@all']],
+            'nullable_type_declaration' => [
+                'syntax' => 'question_mark',
+            ],
+            'nullable_type_declaration_for_default_null_value' => true,
```

Comment:
This [should be an array](https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/blob/master/doc/rules/function_notation/nullable_type_declaration_for_default_null_value.rst) like `['use_nullable_type_declaration' => true]`.  However, it's deprecated and the default value is `true` anyway.

Reply:
I tested this with PHP-CSS-Parser.  It seems that a simple value of `true` does work, presumably for backward-compatibllity.  Also, it seems that in the version of PHP-CS-Fixer we have there (3.59.3, which seems to be the latest), this is NOT the default setting.  So I am a bit mystified.

I am using PHP 7.4.  We have already found that PHP-CS-Fixer behaves differently depending on PHP version in the execution environment.

I think we need to run the fixer with all supported PHP versions and/or get them to sort that problem out.  The execution environment should not affect behaviour.  It's the target environment that matters.

Reply:
Based on my renewed understanding of how the config system works, this change *is* actually correct :)